### ISSUE TITLE
where should i put the text_loader.json?

### ISSUE THREAD

**Issue by mayangzz** [questdb/questdb#1902]
### Describe the bug

I've put the text_loader.json file in /root/.questdb/conf/ directory, and I changed the cairo.sql.copy.formats.file = /root/.questdb/conf/text_loader.json which in server.conf, finally when i start questDB , it appeared can't find the file.
```
    Exception in thread "main" io.questdb.cutlass.json.JsonException: [0] could not find [resource=/root/.questdb/conf/text_loader.json]
at io.questdb@6.2/io.questdb.std.ThreadLocal.initialValue(ThreadLocal.java:36)
at java.base/java.lang.ThreadLocal.setInitialValue(ThreadLocal.java:195)
at java.base/java.lang.ThreadLocal.get(ThreadLocal.java:172)
at io.questdb@6.2/io.questdb.cutlass.json.JsonException.position(JsonException.java:43)
at io.questdb@6.2/io.questdb.cutlass.json.JsonException.$(JsonException.java:39)
at io.questdb@6.2/io.questdb.cutlass.text.types.InputFormatConfiguration.parseConfiguration(InputFormatConfiguration.java:148)
at io.questdb@6.2/io.questdb.PropServerConfiguration.<init>(PropServerConfiguration.java:698)
at io.questdb@6.2/io.questdb.ServerMain.readServerConfiguration(ServerMain.java:513)
at io.questdb@6.2/io.questdb.ServerMain.<init>(ServerMain.java:96)
at io.questdb@6.2/io.questdb.ServerMain.main(ServerMain.java:289)
```

and this picture is my storage localtion.
![image](https://user-images.githubusercontent.com/48407274/154796558-9545f53b-f6e8-4b7f-b0aa-f167526029ab.png)

emmm...
i just want to changed the timestamp format...
because i use curl import csv always appears 
```
invalid timestamp column 'ts'
```

and this is my csv data picture and my request code

![image](https://user-images.githubusercontent.com/48407274/154796808-16fe3de6-2246-43ec-a931-2b435c7d71fc.png)

```
curl -F schema='[{"name": "ts","type": "TIMESTAMP", "pattern": "yyyy-MM-dd HH:mm:ss.SSSUUUNNN"},{"name": "sensitiveWordContent","type": "STRING"},{"name": "sensitiveWordLevel","type": "STRING"},{"name": "sensitiveWordType","type": "STRING"},{"name": "userId","type": "STRING"},{"name": "msgContent","type": "STRING"},{"name": "msgType", "type": "STRING"}]' \
-F data=@sensitiveWord.csv \
-X POST 'http://questdb-test.tideswing.fun/imp?atomicity=1&timestamp=ts&partitionBy=DAY'
```

### Environment

```markdown
- **QuestDB version**: 6,2
- **OS**: Linux
- **Browser**: chrome
```

**Comment by mayangzz** [comment]
Thanks for your response! i used the k3s to deploy QuestDB ,  and i already understand your means, thanks for your answer。Hoping QuestDB can do better in the future。😄

**Comment by glasstiger** [comment]
has been fixed in #1928
